spec/requests/api/v0.0/authentications_spec.rb

```diff
@@ -1,12 +1,14 @@
 RSpec.describe("v0.0 - Authentications") do
+  include ::Spec::Support::TenantIdentity
+
+  let(:headers)         { {"CONTENT_TYPE" => "application/json", "x-rh-identity" => identity} }
   let(:attributes)      { {"username" => "test_name", "password" => "Test Password", "tenant_id" => tenant.id.to_s, "resource_type" => "Tenant", "resource_id" => tenant.id.to_s} }
   let(:collection_path) { "/api/v0.0/authentications" }
-  let(:tenant)          { Tenant.find_or_create_by!(:name => "default", :external_tenant => "external_tenant_uuid")}
 
-  describe("/api/v0.0/authentications") do
+  pending("/api/v0.0/authentications") do
```

Comment:
Interesting, what fails here?

Reply:
That was a local change that shouldn't have been pushed. I've reversed it